.github/workflows/core.yml

```diff
@@ -18,11 +18,15 @@ jobs:
       - name: Setup
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '16.13.1'
```

Comment:
Is there any reason to pin the version other than "best practice"? Or was there an error caused by this? I would like to keep the more generic version if possible. We have also not pinned it in the nvm file and in the package.json